core/domain/config_services.py

```diff
@@ -24,11 +24,21 @@
 
 
 def set_property(committer_id, name, value):
-    """Sets a property value. The property must already be registered."""
+    """Sets a property value. The property must already be registered.
+    
+    Args:
+        commiter_id: str. An id of the committer.
+        name: str. The name of the property.
+        value: str. The value of the property.
+        
+    Raises:
+        Exception: No config property with the specified name is found.
+        Exception: No config property with the specified name can be modified directly.
```

Comment:
Actually -- I think is_directly_settable is obsolete; it's not used at all any more. Can we deprecate it from both config_domain and config_services?